src/plugins/simulator/model/simulator-content.ts

```diff
@@ -111,6 +112,8 @@ export const SimulatorContentModel = TileContentModel
       // nothing to do here
     },
     step() {
+      withoutUndo();
```

Comment:
This might be OK. 

Simulations are one of things that we didn't have a good plan for in the history system. In the case of DataFlow it doesn't record the node values unless it is storing them in a table, so it is pretty safe. But in this case each step is going to be a history event. So we'll have to see how it goes.

The case where I could see this withoutUndo might cause a problem, is if we are showing the diagram view at the same time. Then if user edits the value of a variable in the diagram view, this edit will be undoable, but the not the simulation changes before or after it. If the user hits undo the value of the variable will jump back to its value just before the edit in the diagram view. I think this might be fine, but perhaps it means this variable has an invalid value based on the other variables.

I think it is fine to try `withoutUndo`, users won't be using the diagram view, so they shouldn't have the problem above. Maybe in the future we'll have to come up with some special support for simulations (things that change the state on a repeating timer).